app/renderer/actions/Inspector.js

```diff
@@ -116,10 +128,27 @@ export function unselectHoveredElement (path) {
  * Requests a method call on appium
  */
 export function applyClientMethod (params) {
-  return async (dispatch) => {
+  return async (dispatch, getState) => {
+    let isRecording = params.methodName !== 'quit' &&
+                      params.methodName !== 'source' &&
```

Comment:
The methodName is never set to `source`. The source method is called every time.

If we were to add `source` as a client method call, though, I think we'd want to record it.